yosys-sv/yosys-flow.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This script doesn't make sense to me?

Reply:
I removed this script and moved copying of binary to `build.sh`